test/schema.test.js

```diff
@@ -77,27 +143,171 @@ describe("endpointYamlSchemaV0D1 schema tests", () => {
       "endpoints[1].context must start with a forward slash and can only contain alphanumeric characters, hyphens, and forward slashes.",
       "endpoints[3].context must start with a forward slash and can only contain alphanumeric characters, hyphens, and forward slashes.",
     ];
-    await expectValidationErrors(validateContextFieldYaml, expectedErrors);
+    await expectValidationErrors(
+      ENDPOINTS_YAML,
+      validateContextFieldYaml,
+      expectedErrors
+    );
   });
 
-  test("should fail when context is not provided for endpoint types REST, GraphQl, WS", () => {
-    // have a chat about this with nisrin
-    // about file path to
+  test("should fail when context is not provided for endpoint types REST, GraphQl, WS", async () => {
     const expectedErrors = [
       "endpoints[0].context is required for REST-type endpoints",
       "endpoints[1].context is required for GraphQL-type endpoints",
       "endpoints[2].context is required for WS-type endpoints",
     ];
-    expectValidationErrors(
+    await expectValidationErrors(
+      ENDPOINTS_YAML,
       validateContextRequiredScenariosYaml,
       expectedErrors
     );
   });
 
-  test("should fail when schema file does not exist in the give path", () => {
+  test("should fail when schema file does not exist in the give path", async () => {
     const expectedErrors = [
       "Schema file does not exist at the given path invalid-path.yaml.",
     ];
-    expectValidationErrors(validateSchemaFilePathYaml, expectedErrors);
+    await expectValidationErrors(
+      ENDPOINTS_YAML,
+      validateSchemaFilePathYaml,
+      expectedErrors
+    );
+  });
+});
+
+describe("componentYamlSchemaV1D0 schema tests", () => {
+  test("should validate correctly with valid component.yaml", async () => {
+    const result = await validateComponentYamlSchema(validComponentYaml);
+    expect(result).toBeDefined();
```

Comment:
What is the result when yaml is not validated?

Reply:
Will fail the test with the error,
<img width="1053" alt="Screenshot 2024-10-15 at 8 51 27 AM" src="https://github.com/user-attachments/assets/982169bb-6e5a-4af4-bba7-eb3484f7db86">